invenio_rdm_records/services/schemas/parent/access.py

```diff
@@ -96,7 +96,9 @@ class ParentAccessSchema(Schema, FieldPermissionsMixin):
         # omit fields from dumps except for users with 'manage' permissions
         # allow only 'settings' and 'owned_by'
         "grants": "manage",
+        "grants": "edit",
```

Comment:
the key redefinition in the dict worries me, it seems that the permission levels are working, but code wise it is a little bit suspicious. I would suggest to verify how this dict looks like on the runtime